### ISSUE TITLE
[1.6.4] Latest Version crash on client start

### ISSUE THREAD

**Issue by stidn** [websitenmaker12/Advanced-Genetics#88]
---- Minecraft Crash Report ----
// Don't be sad. I'll do better next time, I promise!

Time: 24/04/14 3:02 PM
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.ClassNotFoundException: com.advGenetics.AdvGenetics
    at cpw.mods.fml.common.LoadController.transition(LoadController.java:156)
    at cpw.mods.fml.common.Loader.loadMods(Loader.java:521)
    at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:183)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:473)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:808)
    at net.minecraft.client.main.Main.main(SourceFile:101)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:131)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:27)
Caused by: java.lang.ClassNotFoundException: com.advGenetics.AdvGenetics
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:186)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at cpw.mods.fml.common.ModClassLoader.loadClass(ModClassLoader.java:61)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Unknown Source)
    at cpw.mods.fml.common.FMLModContainer.constructMod(FMLModContainer.java:462)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventHandler.handleEvent(EventHandler.java:74)
    at com.google.common.eventbus.SynchronizedEventHandler.handleEvent(SynchronizedEventHandler.java:45)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:313)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:296)
    at com.google.common.eventbus.EventBus.post(EventBus.java:267)
    at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:201)
    at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:181)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventHandler.handleEvent(EventHandler.java:74)
    at com.google.common.eventbus.SynchronizedEventHandler.handleEvent(SynchronizedEventHandler.java:45)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:313)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:296)
    at com.google.common.eventbus.EventBus.post(EventBus.java:267)
    at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:112)
    at cpw.mods.fml.common.Loader.loadMods(Loader.java:511)
    ... 10 more
Caused by: java.lang.NullPointerException
    at org.objectweb.asm.ClassReader.<init>(Unknown Source)
    at codechicken.lib.asm.ASMHelper.createClassNode(ASMHelper.java:113)
    at codechicken.lib.asm.ASMHelper.createClassNode(ASMHelper.java:107)
    at codechicken.core.asm.DefaultImplementationTransformer.transform(DefaultImplementationTransformer.java:84)
    at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:274)
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:172)
    ... 38 more
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- System Details --
Details:
    Minecraft Version: 1.6.4
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.8.0, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 129306336 bytes (123 MB) / 372768768 bytes (355 MB) up to 954728448 bytes (910 MB)
    JVM Flags: 2 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    Suspicious classes: FML and Forge are installed
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v8.11 FML v6.4.49.965 Minecraft Forge 9.11.1.965 36 mods loaded, 36 mods active
    mcp{8.09} [Minecraft Coder Pack](minecraft.jar) Unloaded->Constructed
    FML{6.4.49.965} [Forge Mod Loader](minecraftforge-9.11.1.965.jar) Unloaded->Constructed
    Forge{9.11.1.965} [Minecraft Forge](minecraftforge-9.11.1.965.jar) Unloaded->Constructed
    AppliedEnergistics-Core{rv14.finale3} [AppliedEnergistics Core](minecraft.jar) Unloaded->Constructed
    CodeChickenCore{0.9.0.9} [CodeChicken Core](minecraft.jar) Unloaded->Constructed
    NotEnoughItems{1.6.1.9} [Not Enough Items](NotEnoughItems 1.6.1.9.jar) Unloaded->Constructed
    OpenModsCore{0.4} [OpenModsCore](minecraft.jar) Unloaded->Constructed
    cuchaz.ships{0.8 BETA} [Ships Mod](shipsMod-v0.8.beta.jar) Unloaded->Constructed
    TConstruct-Preloader{0.0.1} [Tinkers Corestruct](minecraft.jar) Unloaded->Constructed
    WarpDriveCore{1.0.0.1} [WarpDriveCore](minecraft.jar) Unloaded->Constructed
    DamageIndicatorsMod{2.9.2.3} [Damage Indicators](1.6.4 DamageIndicatorsv2.9.2.3.zip) Unloaded->Constructed
    advancedgenetics{v1.4.2} [Advanced Genetics](advancedgenetics-1.4.2-1.6.zip) Unloaded->Errored
    arsmagica2{1.1.2} [Ars Magica 2](AM2_1.1.2.zip) Unloaded->Constructed
    AppliedEnergistics{rv14.finale3} [Applied Energistics](appeng-rv14-finale3-mc16x.jar) Unloaded->Constructed
    BuildCraft|Core{4.2.2} [BuildCraft](buildcraft-A-1.6.4-4.2.2.jar) Unloaded->Constructed
    BuildCraft|Builders{4.2.2} [BC Builders](buildcraft-A-1.6.4-4.2.2.jar) Unloaded->Constructed
    BuildCraft|Energy{4.2.2} [BC Energy](buildcraft-A-1.6.4-4.2.2.jar) Unloaded->Constructed
    BuildCraft|Factory{4.2.2} [BC Factory](buildcraft-A-1.6.4-4.2.2.jar) Unloaded->Constructed
    BuildCraft|Transport{4.2.2} [BC Transport](buildcraft-A-1.6.4-4.2.2.jar) Unloaded->Constructed
    BuildCraft|Silicon{4.2.2} [BC Silicon](buildcraft-A-1.6.4-4.2.2.jar) Unloaded->Constructed
    ChickenChunks{1.3.3.4} [ChickenChunks](ChickenChunks 1.3.3.4.jar) Unloaded->Constructed
    ComputerCraft{1.63} [ComputerCraft](ComputerCraft1.63.jar) Unloaded->Constructed
    EnderStorage{1.4.3.6} [EnderStorage](EnderStorage 1.4.3.6.jar) Unloaded->Constructed
    IC2{2.0.397-experimental} [IndustrialCraft 2](industrialcraft-2_2.0.397-experimental.jar) Unloaded->Constructed
    OpenMods{0.4} [OpenMods](OpenModsLib-0.4b.jar) Unloaded->Constructed
    OpenBlocks{1.2.7} [OpenBlocks](OpenBlocks-1.2.7.jar) Unloaded->Constructed
    ForgeMultipart{1.0.0.244} [Forge Multipart](ForgeMultipart-universal-1.6.4-1.0.0.244.jar) Unloaded->Constructed
    TConstruct{1.6.X_1.5.5.2} [Tinkers' Construct](TConstruct_mc1.6.4_1.5.5.2.jar) Unloaded->Constructed
    Translocator{1.1.0.15} [Translocator](Translocator 1.1.0.15.jar) Unloaded->Constructed
    WarpDrive{1.2.0} [WarpDrive](WarpDrive.zip) Unloaded->Constructed
    WR-CBE|Core{1.4.0.7} [WR-CBE Core](WR-CBE 1.4.0.7.jar) Unloaded->Constructed
    WR-CBE|Addons{1.4.0.7} [WR-CBE Addons](WR-CBE 1.4.0.7.jar) Unloaded->Constructed
    WR-CBE|Logic{1.4.0.7} [WR-CBE Logic](WR-CBE 1.4.0.7.jar) Unloaded->Constructed
    mod_ZanMinimap{0.9.4} [Zan's Minimap](ZansMinimap1.6.4.zip) Unloaded->Constructed
    McMultipart{1.0.0.244} [Minecraft Multipart Plugin](ForgeMultipart-universal-1.6.4-1.0.0.244.jar) Unloaded->Constructed
    ForgeMicroblock{1.0.0.244} [Forge Microblocks](ForgeMultipart-universal-1.6.4-1.0.0.244.jar) Unloaded->Constructed
    TConstruct Environment: Sane and ready for action. Bugs may be reported.

**Comment by noxx2** [comment]
I'm getting this error as well
**EDIT**: if you use an older version (1.4.1 according to http://ag.teamdna.de/sites/versionOverview.php), it actually works.

**Comment by stidn** [comment]
Yeah, I used 1.1v for 1.6.4 and that seemed to work, it's just the latest that has issues. I'll have to stick with an outdated version till it gets hopefully fixed, a little surprised that not many are complaining about the same issue.

**Comment by speakerrob** [comment]
No, I'm having the same issue as well. I just noticed today none of the recipes were in the game, so I checked my logs and the mod wasn't even being loaded. After checking the site again, to make sure I got the right download, I noticed that my version was a .jar and the newest download was a .zip, but with the exact same version and build number. Weird.

Anyways, after replacing the .jar with the .zip, it did attempt to load this time, but with the error you posted above. I've gone back a version as noxx2 stated and that seems to work so far.
